app.py

```diff
@@ -1,31 +1,33 @@
 from flask import Flask, render_template, request, redirect, url_for
-import session_items as session
+from config import TRELLO_KEY
+import trello
+from status import COMPLETED
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
 # Allowing POST so we don't need frontend JS
-@app.route('/item/<int:id>/mark-completed', methods=['PUT', 'POST'])
+@app.route('/item/<id>/mark-completed', methods=['PUT', 'POST'])
 def mark_completed(id):
-    item = session.get_item(id)
-    item['status'] = "Completed"
-    session.save_item(item)
+    trello.set_status(id, COMPLETED)
     return redirect('/')
 
-@app.route('/item/<int:id>/delete', methods=['DELETE', 'POST'])
+@app.route('/item/<id>/delete', methods=['DELETE', 'POST'])
 def delete(id):
-    item = session.delete_item(id)
+    item = trello.delete_item(id)
     return redirect('/')
 
 @app.route('/item/add', methods=['POST'])
 def add_item():
-    session.add_item(request.form.get('title'))
+    trello.add_item(request.form.get('title'))
     return redirect('/')
 
 @app.route('/')
 def index():
-    items = session.get_items()
-    sorted_items = sorted(items, key=lambda item: 0 if item['status'] == "Not Started" else 1)
+    items = trello.get_items()
+    print([str(item) for item in items])
+    sorted_items = sorted(items, key=lambda item: 0 if not item.is_completed() else 1)
```

Comment:
Don't you just mean `sorted_items = sorted(items, key=lambda item: item.is_completed())` here?